### PR TITLE
Log search New Visualizations - RN

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-service/2024-01-11-search.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: New Visualizations (Search) 
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - log_level
```
Comment (ssharma-sumo) [inline blog-service/2024-01-11-search.md:5]: Should we have keywords related to Aggregate charts here ??

File: blog-service/2024-01-11-search.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We're excited to announce new visualizations for Log Search queries with which you can have a unified experience with the charts across the Sumo Logic platforms. 
+
+Click the **New Visualizations** button to enable new visualizations for your log search.
```
Comment (ssharma-sumo) [inline blog-service/2024-01-11-search.md:18]: We should also include information about to updating the user preference when user toggle between new to old charts phase 1

File: blog-service/2024-01-11-search.md (hunk 1 above)
Comment (kimsauce) [reply blog-service/2024-01-11-search.md:5]: @ssharma-sumo @JV0812 should aggregate charts also be mentioned in the body? It looks like in the screenshot, the "Aggregates" tab is selected.

**Review by kimsauce — COMMENTED** [review]
one comment

File: blog-service/2024-01-11-search.md (hunk 2 above)
Comment (JV0812) [reply blog-service/2024-01-11-search.md]: Content has been updated.

File: blog-service/2024-01-11-search.md (hunk 1 above)
Comment (kimsauce) [reply blog-service/2024-01-11-search.md:5]: @JV0812 bump ☝️

Comment (JV0812) [reply blog-service/2024-01-11-search.md:5]: added Aggregates as a new keyword.

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. However, if you are publishing today, you need to change the date of the release note file to today's date.

**Pull request merged**